### ISSUE TITLE
Inline examples do not use markdown consistently

### ISSUE THREAD

**Issue by malandis** [momentohq/public-dev-docs#123]
Some API reference code examples do not use markdown, eg list concatenate front. Sweep through the API reference and ensure the code examples use the inline code markdown.

**Comment by NoSQLKnowHow** [comment]
This should be fixed with @cprice404's update to the API reference pages to pull from SDK repos. Chris can you pls confirm?

**Comment by cprice404** [comment]
@NoSQLKnowHow yes, I agree, though I still need to file a few more PRs to get to the finish line on that.
